packages/gatsby/package.json

```diff
@@ -76,7 +74,6 @@
     "graphql-tools": "^3.0.4",
     "graphql-type-json": "^0.2.1",
     "hash-mod": "^0.0.5",
-    "history": "^4.6.2",
     "invariant": "^2.2.4",
```

Comment:
we don't need `history` any more, right?

Reply:
aye